repositories.json

```diff
@@ -70,5 +70,6 @@
   "CharGrinn": "https://raw.githubusercontent.com/notsafeadults64/CharGrinn_Modlist/main/modlists.json",
   "High_and_Dry": "https://raw.githubusercontent.com/Jayer117-ball/HighandDry/main/modlists.json",
   "GhoulifiedReality": "https://raw.githubusercontent.com/Ghoulified/Ghoulified-Reality/main/Modlists.json",
-  "Skyrim_REMASTERED": "https://raw.githubusercontent.com/Tpiac/modlists/main/modlists.json"
+  "Skyrim_REMASTERED": "https://raw.githubusercontent.com/Tpiac/modlists/main/modlists.json",
+  "Fear of the Wanderer": "https://raw.githubusercontent.com/StrawChannel95/Fear-of-the-Wanderer-UI-Access/main/fear%20of%20the%20wanderer.json"
```

Comment:
this can't have spaces in them replace them with `_` or use an abbreviation.

also your machineURL needs to be just `fotw` or `fear_of_the_wanderer`. 

@SovnSkyrim the machineURL you use inside of WJ uses the `/` to separate repository name and machineURL, the machine URL you write inside the repository doesn't.